### PR TITLE
Fixes multiline markdown link text

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/idbtransaction/abort/index.md [hunk 1]
```diff
@@ -48,8 +48,7 @@ In the following code snippet, we open a read/write transaction on our database
 some data to an object store. Note also the functions attached to transaction event
 handlers to report on the outcome of the transaction opening in the event of success or
 failure. At the end, we abort any activity done under the current transaction using
-`abort()`. For a full working example, see our [To-do
-Notifications](https://github.com/mdn/to-do-notifications/) app ([view
+`abort()`. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([view
 example live](https://mdn.github.io/to-do-notifications/).)
```
Comment (teoli2003) [inline files/en-us/web/api/idbtransaction/abort/index.md:52]:
```suggestion
`abort()`. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([View example live](https://mdn.github.io/to-do-notifications/)).
```

File: files/en-us/web/api/idbtransaction/error/index.md [hunk 2]
```diff
@@ -36,8 +36,7 @@ some data to an object store. Note also the functions attached to transaction ev
 handlers to report on the outcome of the transaction opening in the event of success or
 failure. Note the `transaction.onerror = function(event) { };` block, making
 use of `transaction.error` to help in reporting what went wrong when the
-transaction was unsuccessful. For a full working example, see our [To-do
-Notifications](https://github.com/mdn/to-do-notifications/) app ([view
+transaction was unsuccessful. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([view
 example live](https://mdn.github.io/to-do-notifications/).)
```
Comment (teoli2003) [inline files/en-us/web/api/idbtransaction/error/index.md:40]:
```suggestion
transaction was unsuccessful. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([View example live](https://mdn.github.io/to-do-notifications/)).
```

File: files/en-us/web/api/idbtransaction/objectstore/index.md [hunk 3]
```diff
@@ -51,8 +51,7 @@ An {{domxref("IDBObjectStore")}} object for accessing an object store.
 In the following code snippet, we open a read/write transaction on our database and add
 some data to an object store. Note also the functions attached to transaction event
 handlers to report on the outcome of the transaction opening in the event of success or
-failure. For a full working example, see our [To-do
-Notifications](https://github.com/mdn/to-do-notifications/) app ([view
+failure. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([view
 example live](https://mdn.github.io/to-do-notifications/).)
```
Comment (teoli2003) [inline files/en-us/web/api/idbtransaction/objectstore/index.md:55]:
```suggestion
failure. For a full working example, see our [To-do Notifications](https://github.com/mdn/to-do-notifications/) app ([View example live](https://mdn.github.io/to-do-notifications/)).
```

File: files/en-us/web/api/mediastreamtrack/applyconstraints/index.md [hunk 4]
```diff
@@ -23,8 +23,7 @@ dimensions, echo cancellation, and so forth.
 Constraints can be used to ensure that the media meets certain guidelines you prefer.
 For example, you may prefer high-density video but require that the frame rate be a
 little low to help keep the data rate low enough not overtax the network. Constraints
-can also specify ideal and/or acceptable sizes or ranges of sizes. See [Applying
-constraints](/en-US/docs/Web/API/Media_Streams_API/Constraints#applying_constraints) in [Capabilities, constraints,
+can also specify ideal and/or acceptable sizes or ranges of sizes. See [Applying constraints](/en-US/docs/Web/API/Media_Streams_API/Constraints#applying_constraints) in [Capabilities, constraints,
 and settings](/en-US/docs/Web/API/Media_Streams_API/Constraints) for more information on how to apply your preferred constraints.
```
Comment (teoli2003) [inline files/en-us/web/api/mediastreamtrack/applyconstraints/index.md:27]:
```suggestion
can also specify ideal and/or acceptable sizes or ranges of sizes.
See [Applying constraints](/en-US/docs/Web/API/Media_Streams_API/Constraints#applying_constraints)
in [Capabilities, constraints, and settings](/en-US/docs/Web/API/Media_Streams_API/Constraints) for more information on how to apply your preferred constraints.
```

File: files/en-us/web/api/notification/click_event/index.md [hunk 5]
```diff
@@ -17,8 +17,7 @@ The **`click`** event of the {{domxref("Notification")}}
 interface fires when the user clicks on displayed {{domxref("Notification")}}.
 
 The default behavior is to move the focus to the viewport of the notification's related
-[browsing
-context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context). If you don't want that behavior, call {{domxref("Event/preventDefault",
+[browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context). If you don't want that behavior, call {{domxref("Event/preventDefault",
   "preventDefault()")}} on the event object.
```
Comment (teoli2003) [inline files/en-us/web/api/notification/click_event/index.md:21]:
```suggestion
[browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context).
If you don't want that behavior, call {{domxref("Event/preventDefault", "preventDefault()")}} on the event object.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few consecutive multiline links were not all fixes.

**Comment by OnkarRuikar** [comment]
I lost my old script. :.(  The new one had the bug.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**